### ISSUE TITLE
Costume Problem - Fall 2015 equipment / gear disappeared after it was bought

### ISSUE THREAD

**Issue by JuniperViking** [HabitRPG/habitica#6155]
Alys told me via the tavern to just make a new post.
I couldn't get the costume even though I first clicked about an hour and a half before midnight EST. 
this:
https://twitter.com/DoranOfTheNorth/status/660658325684645888
resulted in this on refresh:
https://twitter.com/DoranOfTheNorth/status/660658428096983040

ID is 235a1cbd-48c5-41b1-afb4-59d2f8645c57
Sorry. :-/ My guild needs a healer and I also just classed today after rebirth, and I like this healer gear. It fits me, personality wise, and i was using it as motivation for about two weeks now. Heh...

**Comment by Taratard** [comment]
I have been having the same problem. I've been trying since like, 11:20 EST. I've been trying to buy the healer items before they go away, but it keeps refunding all my gold. My id is 1dc84bda-603a-47e6-8575-c20c98ed7f69

**Comment by LostRalph** [comment]
I had the same issue last night at 11:30 PM MST trying to buy the Halloween Mage set before it went away. My ID is 61342bf2-1eba-498e-a958-0cf9452863e1

**Comment by lemoness** [comment]
I'm very sorry that you all experienced this problem! We're investigating it now. Rest assured that you will all get your equipment, even if we have to grant it manually.

**Comment by SabreCat** [comment]
@JuniperViking @Taratard @LostRalph: I've tweaked the availability to catch a few more time zones. Can you refresh the page and have another go at buying the equipment?

**Comment by CookieDoe** [comment]
i also have bought the set only to have it disappear last night with all my gold returned to me. (mage) 
My ID is 606e5699-8694-404c-86ac-663b3daa2051

**Comment by LostRalph** [comment]
I was able to buy it, and keep it. I've refreshed the page, checked off
some habits, gone to other pages, and I still have the mage set.

Thank you for your help.
Ralph

**Comment by SabreCat** [comment]
@CookieDoe: As above, the items have already had their availability extended. If you buy them now you should be set!

**Comment by Taratard** [comment]
I bought the items, and after refreshing and doing a lot of messing around with completing/creating tasks, and it's all staying! Thank you so much! :D

**Comment by CookieDoe** [comment]
how long is the extension?

**Comment by CookieDoe** [comment]
i refreshed it didn't stay

**Comment by CookieDoe** [comment]
where is the wooden plank? I thought that was only 90 coins?

**Comment by CookieDoe** [comment]
bought all again. cast healing on my team. everything left.

**Comment by CookieDoe** [comment]
i've bought everything. cast spells and then it resets.

**Comment by TripleBzzz** [comment]
Alys suggested I post a comment here - my Habitica Id # is a3e90129-ca29-44e7-adcd-12a152d0515d. I tried to buy the Mage's enchantened thread feom my rewards page and it kept giving me the message "you can't buy this item" even though I had enough gold. and there were no other items for sale under gear/costumes

**Comment by Alys** [comment]
@CookieDoe I've given you the Autumn 2015 Warrior weapon, which you were missing.  I've removed some of your recent comments to make it easier to track what's left to be done. :)

@TripleBzzz I've given you the Thread.

If anyone else is still missing equipment that they tried to buy while it was available, please comment here in the next couple of days.

**Comment by JuniperViking** [comment]
I am still missing it. The time zone extension managed to miss me. I refreshed several times during that time, and played quite a bit, but found no ability to buy it. It didn't show up in the list anymore after midnight on the 31st, period, which occurred for me about an hour after my original post.

**Comment by Alys** [comment]
@JuniperViking I've given you the 2015 Fall Healer equipment. Tell me if it doesn't show up after a reload.

If anyone else is still missing equipment that they tried to buy while it was available, please comment here in the next couple of days and state which pieces you are missing, for which class.

**Comment by JuniperViking** [comment]
YAY! I got it! Thank you, thank you!

**Comment by HunterKahn** [comment]
I had the same problem in that I bought the Rouge-class items Bat-tle Wings and right-handed Bat-tle Ax (already own the left-handed ax and the armor) before the Fall Festival ended, but then whenever I'd revisit the site the items would be gone. I tweeted to Habitica about this and they said I should post here with my User ID and that it would be resolved. My User ID is a4215755-f047-4c34-806d-aa04cd5608b9. Let me know if I need to do anything more. Thanks!

**Comment by Alys** [comment]
@HunterKahn That should be all fixed up for you! Tell me if it isn't. Reload first if you don't see them straight away.

**Comment by alentheunique** [comment]
Hi! My ID is 5d2d847f-1fc7-481f-9032-58ec675c7543
I have lost my Limited autumn 2015 weapon (It looks like a rule or a plank), It adds strength, so I think it belongs to warriors? I don't know, I just got to level 10. Thanks!

**Comment by jimrthy** [comment]
Hi there! My ID's db401af4-47d6-4d5c-8e5f-bd6c39d12504. I started out with the bird seed, scarecrow armor, and scarecrow hat. After I lost them the official last night, I found out about the plank. I found out about the extension around 9:30 the next night. I thought I had the plank, hat, and armor then (couldn't afford all 4 thanks to my spiffy potion of rebirth), but they were gone the next day when I logged in.

This is the first time I've had a chance to look back here, and I won't be horribly disappointed if I have to wait until next year, but I figured it's worth asking.

Either way, thanks very much!
